01-import_and_maxfilter.py

```diff
@@ -126,11 +127,47 @@ def run_maxwell_filter(subject, session=None):
         if hasattr(raw, 'fix_mag_coil_types'):
             raw.fix_mag_coil_types()
 
+        if config.find_flat_channels_meg or config.find_noisy_channels_meg:
+            if (config.find_flat_channels_meg and
+                    not config.find_noisy_channels_meg):
+                msg = 'Finding flat channels.'
+            elif (config.find_noisy_channels_meg and
+                  not config.find_flat_channels_meg):
+                msg = 'Finding noisy channels using Maxwell filtering.'
+            else:
+                msg = ('Finding flat channels, and noisy channels using '
+                       'Maxwell filtering.')
+
+            print(msg)
+            raw_lp_filtered_for_maxwell = (raw.copy()
+                                           .filter(l_freq=None,
+                                                   h_freq=40,
+                                                   verbose=True))
+            auto_noisy_chs, auto_flat_chs = find_bad_channels_maxwell(
+                raw=raw_lp_filtered_for_maxwell,
+                calibration=config.mf_cal_fname,
+                cross_talk=config.mf_ctc_fname,
+                verbose=True)
+            del raw_lp_filtered_for_maxwell
+
+            bads = raw.info['bads'].copy()
```

Comment:
raw.info['bads'] can typically be a list. I would do:
```suggestion
            bads = list(raw.info['bads'])
```

Reply:
How's that better? I find `list.copy()` more readable

Reply:
I did not know you could do a .copy() on a list :)

Reply:
It's been possible since Python 3.3 or so :)